core/finality-grandpa/Cargo.toml

```diff
@@ -25,10 +25,10 @@ network = { package = "substrate-network", path = "../network" }
 service = { package = "substrate-service", path = "../service", optional = true }
 srml-finality-tracker = { path = "../../srml/finality-tracker" }
 fg_primitives = { package = "substrate-finality-grandpa-primitives", path = "primitives" }
-grandpa = { package = "finality-grandpa", version = "0.8.1", features = ["derive-codec"] }
+grandpa = { package = "finality-grandpa", path = "/home/marcio/repos/finality-grandpa/", features = ["derive-codec"] }
```

Comment:
This won’t work for anyone but you.  This path should be relative.

Reply:
Yeah, it was depending on (now merged) changes in `Grandpa` repository (and I think `branch` was not working).